### DISCUSSION TITLE
Downloading MIMIC CXR

### DISCUSSION THREAD

**Discussion by ehsanealam** [MIT-LCP/mimic-code#1697] [MIMIC-IV-CXR]
Hello, 
Can anyone help me to download MIMIC CXR to my local Directory. I don't have knowledge about Google Cloud and I also do not wanna use that can anybody  please tell me how to download that?  
Thanks for your time.

**Comment by Asaad-Pak** [comment]
Hi @ehsanealam first of all you have to get access to a dataset from PhysioNet for this you have to fill a form first on the website to become a credentialed user, then you have to do the course and then you have to sign the DUA agreement for access. After this there are three methods that will appear in front of you. You can use the last method starting by "wget" command into your local directory. All the information is present at the end of this page: https://physionet.org/content/mimic-cxr/2.0.0/

**Comment by ehsanealam** [comment]
I already done everything you told me but , my point is I cannot see the
bucket in Google cloud.

**Reply by marymlucas** [reply]
@ehsanealam Did you additionally request access to be able to download it from Google? You can either use BigQuery or Cloud

![image](https://github.com/MIT-LCP/mimic-code/assets/34073785/6289decf-d841-4364-86f8-c2349f44273e)

**Comment by ehsanealam** [comment]
Yes, I did. But, In the cloud there are no Buckets.

**Reply by alistairewj** [reply]
In order to use Google cloud, you must use a valid Google identity in your cloud profile. Some institutes provide GSuite, which means your institutional e-mail is valid, but not all. I would go through these steps again:

1. Set a Google e-mail as your ID in your cloud profile: https://physionet.org/settings/cloud/
    * GSuite IDs or Gmail accounts work
    * If necessary, create a new gmail account. If you do this, you will also need to navigate to https://console.cloud.google.com/ to set up a project ID so that you can pay for your download(s), if any.
2. Request access via the Project page
3. Verify you received an e-mail from PhysioNet confirming you have been added to the access list
4. Verify you can see the dataset bucket, in this case: https://console.cloud.google.com/storage/browser/mimic-cxr-2.0.0.physionet.org
